### PR TITLE
Replace usage of PNOR with Host

### PR DESCRIPTION
**Pull request by gurevichmark**
#4926 

1. Replace usage of PNOR with Host
2. Sync up `.rst` files with `.pod`

### CODE REVIEW

File: docs/source/references/coral/cluster_mgmt/firmware/ipmi.rst [hunk 1]
```diff
@@ -10,7 +10,7 @@ Collect the required files
 Collect the following files and put them into a directory on the Management Node. 
 
    * pUpdate
-   * pnor
```
Comment (whowutwut) [inline docs/source/references/coral/cluster_mgmt/firmware/ipmi.rst]: Is this understandable from the supermicro point of view, what do they name the pnor image file?

**Comment by gurevichmark** [comment]
Updated to make it a little more clear.

**Comment by whowutwut** [comment]
@zet809 how do you feel about this change?

**Comment by zet809** [comment]
Good to me, thx very much @gurevichmark and @whowutwut , let me merge it.

**Pull request merged**